util/cron/nightly

```diff
@@ -822,9 +825,12 @@ if ($runtests == 0) {
             $genGraphsMessage = "Generating release over release performance graphs";
             mysystem($genGraphsCommand, $genGraphsMessage, 0, 1, 1);
 
-            # sync the perf graphs over to sourceforge
-            $rsyncCommand = "$chplhomedir/util/cron/syncPerfGraphs.py $svnReleasePerfDir/html/ $perfConfigName/releaseOverRelease/$syncsuffix --logFile $logdir/syncReleaseToSourceForge.errors";
-            $rsyncMessage = "syncing release performance graph to sourceforge -- log file at $logdir/syncReleaseToSourceForge.errors";
+            # sync the perf graphs over to dreamhost
+            my $str = "syncReleaseToDreamhost.$perfConfigName/releaseOverRelease/$syncsuffix";
+            $str =~ s,/,-,g;
+            $str =~ s,-*$,,;
```

Comment:
This means nothing to me (That may be because I don't know perl, but most of us don't know perl well enough to read this easily)

Reply:
The part I added, attempts to name the "errors" log file after the target path of the file transfer being logged.